src/main/webapp/go.html

```diff
@@ -32,9 +32,10 @@
           <button class="button" id="start-button" type="button" onclick="startHunt()">Start Hunt</button>
           <button class="button" id="hint-button" type="button" onclick="getHint()">Get a Hint</button>
           <button class="button" id="proceed-button" type="button" onclick="proceed()">Proceed to Next Destination</button>
+          <button class="button" id="clear-hint-button" type="button" onclick="deleteMessage(HINT_DISPLAY)">Clear Hints</button>
```

Comment:
Does `deleteMessage(HINT_DISPLAY)` work as expected? I'm not sure, because `HINT_DISPLAY` may not be referenceable from this location, but perhaps it is.

Reply:
yes, this works as expected!